scripts/tutorial.ipynb

```diff
@@ -819,11 +819,266 @@
+       "(1/2) tensor([[0., 0., 0.],\n",
+       "(1/2)         [0., 0., 0.],\n",
+       "(1/2)         [0., 0., 0.],\n",
+       "(1/2)         [0., 0., 0.]])"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "a = ht.zeros((4, 6,), split=1)\n",
+    "a"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(0/2) tensor([[0., 0., 0., 0., 0., 0.],\n",
+       "(0/2)         [0., 0., 0., 0., 0., 0.]])\n",
+       "(1/2) tensor([[0., 0., 0., 0., 0., 0.],\n",
+       "(1/2)         [0., 0., 0., 0., 0., 0.]])"
+      ]
+     },
+     "execution_count": 31,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "a.resplit(0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Distributed operations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(0/2) tensor([3, 4, 5, 6, 7], dtype=torch.int32)\n",
+       "(1/2) tensor([8, 9, 10, 11, 12], dtype=torch.int32)"
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ht.arange(10, split=0) + 3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Operations between tensors with equal split or no split are fully parallelizable and therefore very fast."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(0/2) tensor([1, 2, 3, 4, 5], dtype=torch.int32)\n",
+       "(1/2) tensor([6, 7, 8, 9, 10], dtype=torch.int32)"
+      ]
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "a = ht.arange(10, split=0)\n",
+    "b = ht.ones((10,), split=0)\n",
+    "a + b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The example below will show that it is also possible to use operations on tensors with different split and the proper result calculated. However, this should be used seldomly and with small data amounts only, as it entails sending large data amount over the network."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(0/2) tensor([[9., 9., 9., 9., 9., 9.],\n",
+       "(0/2)         [9., 9., 9., 9., 9., 9.]])\n",
+       "(1/2) tensor([[9., 9., 9., 9., 9., 9.],\n",
+       "(1/2)         [9., 9., 9., 9., 9., 9.]])"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
```

Comment:
This cell yields an error on my system

```
a = ht.full((4, 6,), 8, split=0)

b = ht.ones((4, 6,), split=1)

a + b

---------------------------------------------------------------------------
NotImplementedError                       Traceback (most recent call last)
<ipython-input-20-ad92771abf7e> in <module>
      1 a = ht.full((4, 6,), 8, split=0)
      2 b = ht.ones((4, 6,), split=1)
----> 3 a + b

/p/project/cjsc/bjoernh/Daten/JSC/Development/HAF/venves/heat-p3/lib/python3.6/site-packages/heat/core/dndarray.py in __add__(self, other)
    245                 [5., 6.]])
    246         """
--> 247         return arithmetics.add(self, other)
    248 
    249     def all(self, axis=None, out=None, keepdim=None):

/p/project/cjsc/bjoernh/Daten/JSC/Development/HAF/venves/heat-p3/lib/python3.6/site-packages/heat/core/arithmetics.py in add(t1, t2)
     53 
     54     """
---> 55     return operations.__binary_op(torch.add, t1, t2)
     56 
     57 

/p/project/cjsc/bjoernh/Daten/JSC/Development/HAF/venves/heat-p3/lib/python3.6/site-packages/heat/core/operations.py in __binary_op(operation, t1, t2)
     83                 # It is NOT possible to perform binary operations on tensors with different splits, e.g. split=0
     84                 # and split=1
---> 85                 raise NotImplementedError('Not implemented for other splittings')
     86 
     87             # ToDo: Fine tuning in case of comm.size>t1.shape[t1.split]. Send torch tensors only to ranks, that will hold data.

NotImplementedError: Not implemented for other splittings
```

Reply:
This is indeed true, currently left in there in broken form as the patch will be provided soon